nano/node/bootstrap/bootstrap.cpp

```diff
@@ -865,10 +873,27 @@ bool nano::bootstrap_attempt::lazy_finished ()
 	return result;
 }
 
+bool nano::bootstrap_attempt::lazy_expired ()
+{
+	bool result (false);
```

Comment:
minor: wdyt about calling this `has_expired` for readability?

Reply:
Good idea